base/uk.ac.stfc.isis.ibex.ui.graphing/src/uk/ac/stfc/isis/ibex/ui/graphing/websocketview/MatplotlibToolbar.java

```diff
@@ -0,0 +1,128 @@
+package uk.ac.stfc.isis.ibex.ui.graphing.websocketview;
+
+import java.util.Map;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.ToolBar;
+import org.eclipse.swt.widgets.ToolItem;
+import org.eclipse.wb.swt.ResourceManager;
+
+public class MatplotlibToolbar {
+	private final ToolBar toolBar;
+	
+	private final Map<String, String> buttonIcons = Map.of(
+			"home", "/resources/home.png",
+			"back", "/resources/left-arrow.png",
+			"forward", "/resources/right-arrow.png",
+			"zoom_active", "/resources/zoom_active.png",
+			"zoom_inactive", "/resources/zoom_inactive.png",
+			"pan_active", "/resources/pan_active.png",
+			"pan_inactive", "/resources/pan_inactive.png"
+	);
+	//ResourceManager.getPluginImage("uk.ac.stfc.isis.ibex.ui.graphing", 
+	
+	public final MatplotlibButton homeButton;
+	public final MatplotlibButton backButton;
+	public final MatplotlibButton forwardButton;
+	public final MatplotlibButton panButton;
+	public final MatplotlibButton zoomButton;
+	@SuppressWarnings("unused")
+	// This is only added to the GUI, so not "used" but is still technically used
+	private final ToolItem separator;
```

Comment:
You can just use `var separator = ...` in the constructor body, no need to store this as a class member